packages/vendure-plugin-customer-managed-groups/src/api-extensions.ts

```diff
@@ -29,15 +31,73 @@ export const shopSchema = gql`
     isGroupAdmin: Boolean
   }
 
+  input UpdateCustomerManagedGroupMemberInput {
+    title: String
+    firstName: String
+    lastName: String
+    emailAddress: String
+    addresses: [AddressInput]
+    customerId: ID!
+  }
+
+  input AddressInput {
+    id: ID
+    fullName: String
+    company: String
+    streetLine1: String
+    streetLine2: String
+    city: String
+    province: String
+    postalCode: String
+    countryCode: String
+    phoneNumber: String
+    defaultShippingAddress: Boolean
+    defaultBillingAddress: Boolean
+  }
+
   type CustomerManagedGroupMember {
     customerId: ID!
     title: String
+    addresses: [CustomAddress!]
     firstName: String!
     lastName: String!
     emailAddress: String!
     isGroupAdministrator: Boolean!
   }
 
+  type CustomAddress {
```

Comment:
It's good that we have a separate address type, but can we rename it to `CustomerManagedGroupAddress`, to keep it in scope of the plugin?